artifactory_cleanup/cli.py

```diff
@@ -179,14 +189,14 @@ def main(self):
                 continue
             total_size += summary.artifacts_size
 
-            result["policies"].append(
-                {
-                    "name": summary.policy_name,
-                    "file_count": summary.artifacts_removed,
-                    "size": summary.artifacts_size,
-                }
-            )
-
+            policy = {
+                "name": summary.policy_name,
+                "file_count": summary.artifacts_removed,
+                "size": summary.artifacts_size
+            }
+            if summary.removed_artifacts_list is not None:
+                policy["artifacts_list"] = summary.removed_artifacts_list
```

Comment:
no needs to specify the type 
```suggestion
                policy["artifacts"] = summary.removed_artifacts_list
```